test/bin/lint.sh

```diff
@@ -5,11 +5,31 @@ set -eu
 unset CDPATH
 cd "$( dirname "${BASH_SOURCE[0]}" )/../.."
 
+echo "Checking for files with DOS encoding:"
+(! git ls-files | xargs file | grep -I "with CRLF line terminators")
+
+echo "Checking for files with windows-style newlines:"
+(! git ls-files | xargs grep -I $'\r')
+
+echo $?
+
+echo keeps running
+
+exit 0
+
 echo "Running pylint ..."
 # TODO: Enable Refactor and Convention reports
-pylint --reports=no --disable=C,R "$@"
+pylint --reports=no --disable=C,R api
+
+echo "Checking for files with DOS encoding:"
+! find * -path "runtime" -prune -o -path "persistent" -prune -o \
+  -type f -exec file {} \; | grep -I "with CRLF line terminators"
+
+echo "Checking for files with windows style newline:"
+! find * -path "runtime" -prune -o -path "persistent" -prune -o -type f \
+  -exec grep -rI $'\r' {} \+
```

Comment:
Looks like this file didn't merge well and still has some debug statements in it.